Twig/RollbarExtension.php

```diff
@@ -91,15 +91,70 @@ public function getInitRollbarCode(array $context)
+{$checkIgnoreConfig}
 </script>
 END_HTML;
     }
 
     /**
+     * @return string
+     */
+    protected function getCheckIgnoreConfig()
+    {
+        $allowedHosts = [];
+        if (
+            isset($this->notifierOptions['allowed_js_hosts']) &&
+            is_array($this->notifierOptions['allowed_js_hosts'])
```

Comment:
I guess that these checks and normalization is better to move into `DependencyInjection/Configuration`